### ISSUE TITLE
Training README datasets table: text column and id column

### ISSUE THREAD

**Issue by guynich** [huggingface/distil-whisper#97]
To pseudo-label the three open-source datasets I had to re-order the [table data](https://github.com/huggingface/distil-whisper/blob/main/training/README.md#1-pseudo-labelling) in the Text Column and ID Column.
```
| Dataset                                                                                       | Languages | Domain                                | Speaking Style | License   | Text Column  | ID Column    |
|-----------------------------------------------------------------------------------------------|-----------|---------------------------------------|----------------|-----------|--------------|--------------|
| [Multilingual LibriSpeech](https://huggingface.co/datasets/facebook/multilingual_librispeech) | 6         | Audiobooks                            | Narrated       | CC-BY-4.0 | `"text"`     | `"id"`       |
| [Common Voice 13](https://huggingface.co/datasets/mozilla-foundation/common_voice_13_0)       | 108       | Wikipedia text & crowd-sourced speech | Narrated       | CC0-1.0   | `"sentence"` | `"path"`     |
| [VoxPopuli](https://huggingface.co/datasets/facebook/voxpopuli)                               | 15        | European Parliament recordings        | Spontaneous    | CC0       | `"raw_text"` | `"audio_id"` |
```

**Comment by guynich** [comment]
For VoxPopuli I found "raw_text" to contain empty strings and think that is why the psuedo-labelling script to fail after hours of compute with a `ValueError("one or more references are empty strings")`.

**Comment by guynich** [comment]
Copied from #98 
When pseudo-labelling the Voxpopuli dataset the "raw_text" (needed for option  --text_column_name) may be an empty string for some examples - see [HF dataset model card here](https://huggingface.co/datasets/facebook/voxpopuli#data-instances) for an empty "raw_text" example.

Question: how do I check which text name ("raw_text" or "normalized_text") was used when creating the pseudo-labelled datasets on HF, such as https://huggingface.co/datasets/distil-whisper/voxpopuli ?

**Comment by sanchit-gandhi** [comment]
Hey @guynich - the provided transcriptions in the original VoxPopuli dataset are only used for computing the WER in the pseudo-labelling and distillation scripts. Since the WER is computed on normalised transcriptions, you can safely use the `"normalized_text"` column in the dataset, which is what was done for the Distil-Whisper datasets.

**Comment by guynich** [comment]
Thank you for the helpful comment and for the fix #102.  Closing.
